### ISSUE TITLE
Training log for few-shot settings

### ISSUE THREAD

**Issue by hungphongtrn** [Flame-Chasers/TBPS-CLIP#6]
First, thank the team for the amazing works and congratulate on your acceptance to AAAI.

I am following TBPS-CLIP on a training setting similar to CUHK-PEDES at 10%. Here are the dataset statistics:
| subset |  ids | images | captions |
|:------:|:----:|:------:|:--------:|
|  train | 2000 |   4302 |     8604 |
|   test | 1000 |   2000 |     4000 |

I am following the settings of using ['SS', 'MVS', 'RITC', 'CITC', 'NITC'] as the loss functions.  However, the loss is already low (at around 8.1) at the end of the first epoch while metrics look really bad at less than 10% for rank 1. 

Can you share the training log so that I could use it as a reference? And if possible, can you give an advice on how could the model performance be this bad?

Thank you in advance!

**Comment by zhendekeng** [comment]
@hungphongtrn Have you successfully reproduced the experimental results of the full data?

**Comment by hungphongtrn** [comment]
@zhendekeng I haven't since I only have a single GPU

**Comment by zhendekeng** [comment]
- @hungphongtrn So your batch size is 80, epoch is 5, and learning rate is 1e-5? This training setting will be underfitting

**Comment by hungphongtrn** [comment]
> - @hungphongtrn So your batch size is 80, epoch is 5, and learning rate is 1e-5? This training setting will be underfitting

Since I only have 12GB VRAM on a single GPU, my setting is batch size of 8/16, 5 epochs, and then the lr will be 1e-5. I only test the scenario of 10% of the CUHK-PEDES, and reached around 43-46% rank 1 (Their report score is around 50%).

**Comment by zhendekeng** [comment]
> > * @hungphongtrn So your batch size is 80, epoch is 5, and learning rate is 1e-5? This training setting will be underfitting
> 
> Since I only have 12GB VRAM on a single GPU, my setting is batch size of 8/16, 5 epochs, and then the lr will be 1e-5. I only test the scenario of 10% of the CUHK-PEDES, and reached around 43-46% rank 1 (Their report score is around 50%).

43-46% rank 1 is a normal result because your batch size is too small

**Comment by hungphongtrn** [comment]
> > > * @hungphongtrn So your batch size is 80, epoch is 5, and learning rate is 1e-5? This training setting will be underfitting
> 
> > 
> 
> > Since I only have 12GB VRAM on a single GPU, my setting is batch size of 8/16, 5 epochs, and then the lr will be 1e-5. I only test the scenario of 10% of the CUHK-PEDES, and reached around 43-46% rank 1 (Their report score is around 50%).
> 
> 
> 
> 43-46% rank 1 is a normal result because your batch size is too small

I only trained with 10% of CUHK-PEDES though

**Comment by hungphongtrn** [comment]
> - @hungphongtrn So your batch size is 80, epoch is 5, and learning rate is 1e-5? This training setting will be underfitting

And you are saying that with a larger batch size, I should use larger lr right?

**Comment by zhendekeng** [comment]
> > * @hungphongtrn So your batch size is 80, epoch is 5, and learning rate is 1e-5? This training setting will be underfitting
> 
> And you are saying that with a larger batch size, I should use larger lr right?

You should adjust the parameters in the full data setting and run the results with the same parameters on 10% of the data

**Comment by hungphongtrn** [comment]
> > > * @hungphongtrn So your batch size is 80, epoch is 5, and learning rate is 1e-5? This training setting will be underfitting
> > 
> > 
> > And you are saying that with a larger batch size, I should use larger lr right?
> 
> You should adjust the parameters in the full data setting and run the results with the same parameters on 10% of the data

Let me try it. It's just training with full data takes forever each time

**Comment by zhendekeng** [comment]
> You should adjust the parameters in the full data setting

You can then adjust the parameters for 10% of the data.

> > > > * @hungphongtrn So your batch size is 80, epoch is 5, and learning rate is 1e-5? This training setting will be underfitting
> > > 
> > > 
> > > And you are saying that with a larger batch size, I should use larger lr right?
> > 
> > 
> > You should adjust the parameters in the full data setting and run the results with the same parameters on 10% of the data
> 
> Let me try it. It's just training with full data takes forever each time

You can then adjust the parameters for 10% of the data.
